### PR TITLE
Improve mediation ui

### PR DESCRIPTION
**Pull request by ripcurlx**
Fixes https://github.com/bisq-network/bisq/pull/3221#issuecomment-530271330, https://github.com/bisq-network/bisq/pull/3221#issuecomment-530288809, https://github.com/bisq-network/bisq/pull/3221#issuecomment-530525346 and https://github.com/bisq-network/bisq/pull/3221#issuecomment-530526358.

### CODE REVIEW

**Comment by ripcurlx** [comment]
@chimp1984 I've adapted your code to address these issues.

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -760,9 +760,15 @@ portfolio.pending.support.headline.getHelp=Need help?
...
   If your issue still isn't resolved, you can request more help from a mediator.
+portfolio.pending.support.text.getHelp.arbitrator=If you have any problems you can try to contact the trade peer in the trade \
+  chat or ask the Bisq community at https://bisq.community. \
+  If your issue still isn't resolved, you can request more help from an arbitrator.
 portfolio.pending.support.button.getHelp=Get support
 portfolio.pending.support.popup.info=If your issue with the trade remains unsolved, you can open a support \
-  ticket to request help from the mediator. If you have not received the payment, please wait until the trade period is over.\n\n\
+  ticket to request help from a mediator. If you have not received the payment, please wait until the trade period is over.\n\n\
+  Are you sure you want to open a support ticket?
+portfolio.pending.support.popup.info.aribtrator=If your issue with the trade remains unsolved, you can open a support \
```
Comment (sqrrm) [inline core/src/main/resources/i18n/displayStrings.properties]: Misspelled `aribtrator`

**Review by sqrrm — CHANGES_REQUESTED** [review]
Please fix spelling mistake, otherwise good

**Review by chimp1984 — APPROVED** [review]
utACK

**Comment by ripcurlx** [comment]
> Please fix spelling mistake, otherwise good

Thanks for spotting the typo. Just pushed the change.

**Pull request merged**